mage_ai/authentication/providers/okta.py

```diff
@@ -36,33 +36,32 @@ def __validate(self):
                 'Okta client secret is empty. '
                 'Make sure the OKTA_CLIENT_SECRET environment variable is set.')
 
-    def get_auth_url_response(self, redirect_uri: str = None, **kwargs) -> Optional[Dict]:
-        if OKTA_CLIENT_ID:
-            base_url = get_base_url(redirect_uri)
-            redirect_uri_query = dict(
-                provider=self.provider,
-                redirect_uri=redirect_uri,
-            )
-            query = dict(
-                client_id=OKTA_CLIENT_ID,
-                redirect_uri=urllib.parse.quote_plus(
-                    f'{base_url}/oauth',
-                ),
-                response_mode='query',
-                response_type='code',
-                scope='openid email profile',
-                state=uuid.uuid4().hex,
-            )
-            query_strings = []
-            for k, v in query.items():
-                query_strings.append(f'{k}={v}')
+    def get_auth_url_response(self, redirect_uri: str = None, **kwargs) -> Dict:
```

Comment:
This is a comment for the `__init__` method in this class. You might need to call `__validate` before settings the self.hostname.